src/hooks.ts

```diff
@@ -35,6 +35,8 @@ type HookOptions = {
   timeout?: number;
 };
 
+type decideHooksOptions = HookOptions & { decideOptions?: OptimizelyDecideOption[] };
```

Comment:
Shouldn't it be uppercase to be consistent? "DecideHooksOptions"

Reply:
Good catch